modules/ossm-vs-istio.adoc

```diff
@@ -10,27 +10,44 @@ An installation of {ProductName} differs from upstream Istio community installat
+
+* *`NetworkPolicy`*: {ProductName} creates a `NetworkPolicy` resource in each member project allowing ingress to all pods from the other members and the control plane. If you remove a member from {ProductShortName}, this `NetworkPolicy` resource is deleted from the project.
++
 [NOTE]
 ====
-Single-tenant control plane installations are known to cause issues with {product-title} restarts and upgrades. Multi-tenant control plane installations are the default configuration starting with {ProductName} {ProductVersion}.
+This also restricts ingress to only member projects. If ingress from non-member projects is required, you need to create a `NetworkPolicy` to allow that traffic through.
 ====
 
-{ProductName} allows you to configure multi-tenant control plane installations, specify the namespaces that can access its {ProductShortName}, and isolate the {ProductShortName} from other control plane instances.
+* *Multitenant*: {ProjectName} joins the `NetNamespace` for each member project to the `NetNamespace` of the control plane project (for example, invoking oc `adm pod-network join-projects --to control-plane-project member-project`). If you remove a member from the {ProductShortName}, its `NetNamespace` is isolated from the control plane (for example, invoking `oc adm pod-network isolate-projects member-project`).
+
+* *Subnet*: No additional configuration is performed.
 
 [id="ossm-automatic-injection_{context}"]
 == Automatic injection
-The upstream Istio community installation automatically injects the sidecar to namespaces you have labeled.
+The upstream Istio community installation automatically injects the sidecar into pods within the projects you have labeled.
 
-{ProductName} does not automatically inject the sidecar to any namespaces, but requires you to specify the `sidecar.istio.io/inject` annotation as illustrated in the xref:../service_mesh_install/prepare-to-deploy-applications-ossm.html#ossm-automatic-sidecar-injection_deploying-applications-ossm[Automatic sidecar injection] section.
+{ProductName} does not automatically inject the sidecar to any pods, but requires you to specify the `sidecar.istio.io/inject` annotation as illustrated in the xref:../service_mesh_day_two/prepare-to-deploy-applications-ossm.adoc#ossm-automatic-sidecar-injection_deploying-applications-ossm[Automatic sidecar injection] section.
 
 [id="ossm-rbac_{context}"]
 == Role Based Access Control features
 Role Based Access Control (RBAC) provides a mechanism you can use to control access to a service. You can identify subjects by username or by specifying a set of properties and apply access controls accordingly.
```

Comment:
I believe this needs to be "Istio Role Based Access Control" to distinguish it from Kubernetes RBAC, which controls access to the API Server (whereas Istio RBAC controls access to services, as this sentence explains).

Reply:
Fixed.